### ISSUE TITLE
Router doesn't render over the parent element

### ISSUE THREAD

**Issue by pukhalski** [aksonov/react-native-router-flux#2218]
### Version

- react-native-router-flux v4.0.0-beta.16
- react-native v0.46.4

### Expected behaviour

Background image of parent component wrapper is visible

### Actual behaviour

Background image of parent component wrapper is not visible

```
<Provider store={store}>
    <ImageBackground source={require('./bg.png')} style={{ position: 'absolute', top: 0, left: 0, right: 0, bottom: 0 }}>
      <Router getSceneStyle={() => ({ backgroundColor: 'transparent' )}>
          <Scene key="home"  component={Home} title="Welcome" />
      </Router>
    </ImageBackground>
  </Provider>
```

**Comment by pukhalski** [comment]
Made it work with a nasty trick:
```
<ImageBackground source={require('./Background.png')} style={style.bg}>
        <View style={{
          position: 'absolute',
          backgroundColor: 'transparent',
          width: '100%' }}>
            ...Router
        </View>
      </ImageBackground>
```

**Comment by devvtms** [comment]
simple try this:
```js
<Image
    style={{
      flex: 1,
      width: null,
      height: null,
      resizeMode: 'cover',
    }}
    source={{
      uri:'',
    }}
  >
<Router getSceneStyle={() => ({ backgroundColor: 'transparent' )}>
          <Scene key="home"  component={Home} title="Welcome" />
      </Router>
</Image>
```
for me working

**Comment by pukhalski** [comment]
@devvtms, thanks! any ideas on how to fix #2220 by a chance? :) 

I need it only for some part of my scenes, not for all of them.

**Comment by aksonov** [comment]
Closing it in favour to #2220
